src/main/java/eu/tsystems/mms/tic/testerra/plugins/selenoid/webdriver/SelenoidCapabilityProvider.java

```diff
@@ -45,16 +43,16 @@
  *
  * @author Eric Kubenka
  */
-public class SelenoidCapabilityProvider implements Consumer<WebDriverRequest>, Loggable {
+public class SelenoidCapabilityProvider implements Consumer<WebDriverRequest>, Loggable, PropertyManagerProvider {
 
     private final boolean VIDEO_ACTIVE = Testerra.Properties.SCREENCASTER_ACTIVE.asBool();
-    private final boolean VNC_ACTIVE = PropertyManager.getBooleanProperty(SelenoidProperties.VNC_ENABLED, SelenoidProperties.Default.VNC_ENABLED);
-    private final String VNC_ADDRESS = PropertyManager.getProperty(SelenoidProperties.VNC_ADDRESS, SelenoidProperties.Default.VNC_ADDRESS);
+    private final boolean VNC_ACTIVE = PROPERTY_MANAGER.getBooleanProperty(SelenoidProperties.VNC_ENABLED, SelenoidProperties.Default.VNC_ENABLED);
+    private final String VNC_ADDRESS = PROPERTY_MANAGER.getProperty(SelenoidProperties.VNC_ADDRESS, SelenoidProperties.Default.VNC_ADDRESS);
 
-    private static final int VIDEO_FRAMERATE = PropertyManager.getIntProperty(SelenoidProperties.VIDEO_FRAMERATE, SelenoidProperties.Default.VIDEO_FRAMERATE);
+    private static final long VIDEO_FRAMERATE = PROPERTY_MANAGER.getLongProperty(SelenoidProperties.VIDEO_FRAMERATE, SelenoidProperties.Default.VIDEO_FRAMERATE);
```

Comment:
Isn't that... a bit much framerate? I mean, at 60 Hz we are still in single byte territory. What would `9 * 10^19` Hz even look like (other than... an explosion, I guess)?

Reply:
But we do not have a `PROPERTY_MANAGER.getByte()` method.

Reply:
Ah, I see. Well, then. It's all about Thinking Big, right? No love for small values in a big(Int) world.